code/game/objects/machinery/deployable.dm

```diff
@@ -64,8 +66,9 @@
 	return ..()
 
 /obj/machinery/deployable/examine(mob/user)
+	if(examine_internal_item)
+		return internal_item.examine(user)
 	. = ..()
-	. += internal_item.examine(user)
```

Comment:
Doesn't it already copy the examine text or some wack stuff, which is why most deployables have a double up?